### ISSUE TITLE
Implementing multi-label classification for RNN classifiers

### ISSUE THREAD

**Issue by DavidWBressler** [fastai/fastai#398]
It would be great if the fast.ai library was set up to automatically handle multi-label classification data for RNN classifiers (as is the case for image classification).

Currently for image classification, ImageClassifierData will automatically recognize multi-label classification data if using from_csv. If the target has more than one column, it defines as true the property 'is_multi'. Then in ConvLearner, the last layer is changed to nn.Sigmoid (instead of nn.LogSoftmax()), and the loss is set to F.binary_cross_entropy (instead of F.nll_loss). 

I propose to implement something similar for RNN classification.

@jph00 is this something that you'd be interested in including in the fast.ai library? Can I try making a PR with an implementation?

**Comment by jph00** [comment]
PRs certainly welcome!
